src/pages/MapPage/components/Filter/FilterDrawer.js

```diff
@@ -96,7 +101,6 @@ const FilterDrawer = props => {
   return (
     <Drawer
       anchor={isWidthUp('sm', width) ? 'left' : 'bottom'}
-      variant='persistent'
       open={open}
       onClose={onClose}
```

Comment:
Please change onClose here to handleCloseFilter

Reply:
Already change